### ISSUE TITLE
WINE Testing Results

### ISSUE THREAD

**Issue by MystiK791** [MBU-Team/OpenMBU#135]
I wanted to see if I could get this game working on Linux under WINE. I installed DirectX9 and launched the game. It booted up to the Garagegames logo, then to the Torque logo, and then proceeded to crash with this error:
![image](https://user-images.githubusercontent.com/69944483/192323542-922ddc40-7913-4055-b45f-aaabdc2f11cd.png)

Trying to launch under Proton with Steam, the game opened a window and immediately crashed.

In conclusion, Linux support through WINE currently does not work.

**Comment by MystiK791** [comment]
I realized I forgot this issue existed, sorry!

I'll go ahead and close it for now, and re-open it if I have any further issues.

Thanks!
